pybaseball/datahelpers/postprocessing.py

```diff
@@ -62,3 +63,33 @@ def convert_percentages(data: pd.DataFrame, columns: List[str]) -> pd.DataFrame:
             pass
 
     return data
+
+
+
+
+def check_is_zero_one(instance, attribute, value):
+    if value not in [0, 1]:
+        raise ValueError("{} must be either 0 or 1, not {}".format(attribute, value))
```

Comment:
`f"{attribute} must be either 0 or 1, not {value}"`?

Reply:
done